boot/core/src/boot/task_helpers/notify.clj

```diff
@@ -26,25 +28,51 @@
   [s]
   (= 0 (:exit (shell/sh "sh" "-c" (format "command -v %s" s)))))
 
+(defn- escape [s]
```

Comment:
Ok maybe it does not escape, I actually never tried this way, sorry for my ignorance, is there a need to explicitely escape a string? Can't we call `pp-str` directly and print to out?